File: 1689-partitioning-into-minimum-number-of-deci-binary-numbers/1689-partitioning-into-minimum-number-of-deci-binary-numbers.cs
public class Solution {
    public int MinPartitions(string n) {
        int sum = 0, mx = 0;
        
        for(int i = n.Length - 1; i >= 0; i--){
            int value = n[i]-'0';

            mx = Math.Max(mx, value);
        }
        return  mx;
    }
}

File: 1689-partitioning-into-minimum-number-of-deci-binary-numbers/1689-partitioning-into-minimum-number-of-deci-binary-numbers.cpp
class Solution {
public:
    int minPartitions(string n) {
        int sum = 0, mx = 0;
        
        for(int i = n.size() - 1; i >= 0; i--){
            int value = n[i]-'0';

            mx = max(mx, value);
        }
        return  mx;
    }
};